Snowdog_Components/components/Atoms/typography/error/_error.scss

```diff
@@ -33,7 +33,7 @@ div {
             }
         }
 
-        .checkbox__label {
+        > .checkbox__label {
```

Comment:
Did You check this change doesn't break anything? Do we always have `.checkbox__label` direct under that `div`?

Reply:
I checked. It's always a direct child of that `div`. Not always directly under it but this is why I used `>` selector instead of `~`. While checking I noticed a few required checkboxes that didn't have `required` classes so I added them.